mlflow/store/abstract_store.py

```diff
@@ -206,6 +207,7 @@ def search_runs(self, experiment_ids, search_filter, run_view_type):
         :param search_filter: :py:class`mlflow.utils.search_utils.SearchFilter` object to encode
             search expression or filter string
         :param run_view_type: ACTIVE, DELETED, or ALL runs
+        :param max_results: Maximum number of runs desired
```

Comment:
Would be nice to add an analogous doc to `list_run_infos` (state that we'll restrict the number of returned runs), unless we plan to remove that API entirely